### PR TITLE
Add bam2fastq fastq header to be recognized

### PR DESCRIPTION
**Pull request by ingkebil**
Might fix the fastq header parsing for fastqs created with bam2fastq. (Can someone elaborate how the fastq files are generated?)

Not tested! Need a file to test with :)

@dnil

### CODE REVIEW

File: cg/meta/analysis.py [hunk 1]
```diff
@@ -185,6 +185,10 @@ def _fastq_header(line):
             rs['lane'] = parts[3]
             rs['flowcell'] = parts[2]
             rs['readnumber'] = parts[6].split(' ')[-1]
+        if len(parts) == 7: # @ST-E00201:173:HCLCGALXX:1:2106:22516:34834/1
```
Comment (dnil) [inline cg/meta/analysis.py]: Looks good to me! :+1:

**Comment by dnil** [comment]
Now fingers crossed this solves the whole issue - at one point MIP did not fetch this info from CG but tried to detect filter status and read direction ("read number" here) directly from fastq.

**Comment by ingkebil** [comment]
I don't know that. As far as I know, read direction always has been taken from the fastq file name. Let me know if you've got files ready for testing :)

**Comment by emiliaol** [comment]
Here are files to test on:
Should be linked to ACC3316A10

/mnt/hds/proj/INBOX/P2262/P2262_101.clean.dedup_1.fastq.gz
/mnt/hds/proj/INBOX/P2262/P2262_101.clean.dedup_2.fastq.gz

**Comment by ingkebil** [comment]
Can you give me files that I can run `cg analysis link` on instead?

**Comment by emiliaol** [comment]
They are now available in the HK bundle ACC3316A10

**Comment by ingkebil** [comment]
Before:
```$ cg analysis link ACC3316A10
ACC3316A10: link FASTQ files
TypeError: int() argument must be a string, a bytes-like object or a number, not 'NoneType'
```

After:
```$ cg analysis link ACC3316A10
ACC3316A10: link FASTQ files
linking: /mnt/hds/proj/INBOX/P2262/P2262_101.clean.dedup_1.fastq.gz -> /mnt/hds/proj/bioinfo/families/suitedtarpon/wgs/ACC3316A10/fastq/1_171015_HHVN3CCXX_ACC3316A10_XXXXXX_1.fastq.gz
linking: /mnt/hds/proj/INBOX/P2262/P2262_101.clean.dedup_2.fastq.gz -> /mnt/hds/proj/bioinfo/families/suitedtarpon/wgs/ACC3316A10/fastq/1_171015_HHVN3CCXX_ACC3316A10_XXXXXX_2.fastq.gz
```

**Pull request merged**